content/en/real_user_monitoring/android/mobile_vitals.md

```diff
@@ -17,13 +17,15 @@ Real User Monitoring offers Mobile Vitals, a set of metrics inspired by [Android
 
 You can view Mobile Vitals for your application by navigating to  **UX Monitoring > Performance Summary** and selecting your application.
 
-{{< img src="real_user_monitoring/android/android_mobile_vitals-1.png" alt="Mobile Vitals in the RUM Explorer" style="width:90%;">}}
+{{< img src="real_user_monitoring/android/android_mobile_vitals-2.png" alt="Mobile Vitals in the RUM Explorer" style="width:90%;">}}
```

Comment:
shouldn't this be more a screenshot like this one? just like on the iOS page, to match the text

<img width="1351" alt="Screenshot 2023-06-21 at 12 22 50" src="https://github.com/DataDog/documentation/assets/53906368/3bba8458-4639-4dff-9927-2ba74530cb16">